tv.kodi.Kodi.json

```diff
@@ -1,62 +1,63 @@
 {
     "app-id": "tv.kodi.Kodi",
     "runtime": "org.freedesktop.Platform",
-    "runtime-version": "1.6",
+    "runtime-version": "18.08",
     "sdk": "org.freedesktop.Sdk",
     "sdk-extensions": [
-        "org.freedesktop.Sdk.Extension.openjdk9"
+        "org.freedesktop.Sdk.Extension.openjdk10"
     ],
     "finish-args": [
         "--device=dri",
-        "--filesystem=home",
+        "--filesystem=xdg-music",
+        "--filesystem=xdg-pictures",
+        "--filesystem=xdg-videos",
```

Comment:
These can probably be read only? `xdg-videos:ro`

Reply:
@TingPing Kodi actually can create new files there, e.g. TV recording or grabbing subtitles.

Reply:
library sources must be writeable